app/renderer/js/electron-bridge.js

```diff
@@ -0,0 +1,44 @@
+const events = require('events');
+const { ipcRenderer } = require('electron');
+
+// we have and will have some non camelcase stuff
+// while working with zulip so just turning the rule off
+// for the whole file.
+/* eslint-disable camelcase */
+class ElectronBridge extends events {
+	// eslint-disable-next-line no-useless-constructor
+	constructor() {
```

Comment:
Minor Thing, can you remove this thing, this is not required anymore.

Reply:
Thanks, done!